packages/cli/src/commands/create/index.ts

```diff
@@ -261,7 +268,12 @@ function updatePackageJson(dest: string, projectName: string, flags: CreateOptio
   updateWorkspaceVersions(pkgJson.dependencies, TUTORIALKIT_VERSION);
   updateWorkspaceVersions(pkgJson.devDependencies, TUTORIALKIT_VERSION);
 
-  fs.writeFileSync(pkgPath, JSON.stringify(pkgJson, undefined, 2));
+  if (provider === 'Netlify' || provider === 'Cloudflare') {
```

Comment:
This is only for Cloudflare: https://tutorialkit.dev/guides/deployment/#headers-configuration

```suggestion
  if (provider.toLowerCase() === 'cloudflare') {
```